### ISSUE TITLE
Issue with HighSierra 10.13.4

### ISSUE THREAD

**Issue by mhamzas** [julian-poidevin/MBPMid2010_GPUFix#65]
Julian, The program is not working for 10.13.4.
It is not able to load the kext file. I've tried the file provided earlier but still it asks for the location of the file, which I am not able to select.

Any solution for this ?

**Comment by mhamzas** [comment]
I've tried but still getting error that file not found and even not able to select it.
**Suggestion :** Include this kext file with new version may be 0.6.1.
Check if file exist, then just run patch otherwise put the file in extensions before running patch.

It'll be easier for others too.

**Comment by julian-poidevin** [comment]
Nice suggestion, thanks

**Comment by zoharargaman** [comment]
Resetting the SMC helped me get 10.13.4 in order

**Comment by jurgenkoppen** [comment]
Hi Julian, I just ran your patch on my 2010 MBP 2.4GHz i5 and it told me that is successfully installed the patch. However, shortly after the machine did another random shutdown. I might also mention some background:  that I had  previously replaced the cap in the GPU rail, actually twice now. It seemed to work well after the first replacement (with a cap I got from Louis Rossmann) until a few weeks later, then crash. So I doubted my solder job , installed another new cap. Again , good for a while on a fresh El Capitan install. But when I put the original SSD ( belonging to my friend for whom I am trying to fix this) the shutdown reoccurred.  I have the panic report if that would help to shed light on this.
